samples/bookinfo/src/productpage/productpage.py

```diff
@@ -59,11 +59,11 @@
 
 servicesDomain = "" if (os.environ.get("SERVICES_DOMAIN") is None) else "." + os.environ.get("SERVICES_DOMAIN")
 detailsHostname = "details" if (os.environ.get("DETAILS_HOSTNAME") is None) else os.environ.get("DETAILS_HOSTNAME")
-detailsPort = "9080" if (os.environ.get("DETAILS_PORT") is None) else os.environ.get("DETAILS_PORT")
+detailsPort = "9080" if (os.environ.get("DETAILS_APP_PORT") is None) else os.environ.get("DETAILS_APP_PORT")
```

Comment:
Why aren't you just using `DETAILS_SERVICE_PORT` which is the actual port of the details `Service`?
```suggestion
detailsPort = "9080" if (os.environ.get("DETAILS_SERVICE_PORT") is None) else os.environ.get("DETAILS_SERVICE_PORT")
```

Reply:
makes sense